files/en-us/web/html/element/hgroup/index.html

```diff
@@ -113,22 +116,26 @@ <h2 id="Examples">Examples</h2>
 <h2 id="Specifications">Specifications</h2>
 
 <table class="standard-table">
- <thead>
+  <thead>
   <tr>
-   <th scope="col">Specification</th>
-   <th scope="col">Status</th>
-   <th scope="col">Comment</th>
+    <th scope="col">Specification</th>
+    <th scope="col">Status</th>
+    <th scope="col">Comment</th>
   </tr>
- </thead>
- <tbody>
+  </thead>
+  <tbody>
   <tr>
-   <td>{{SpecName('HTML WHATWG', 'semantics.html#the-hgroup-element', '&lt;hgroup&gt;')}}</td>
-   <td>{{Spec2('HTML WHATWG')}}</td>
-   <td></td>
+    <td>{{SpecName('HTML WHATWG', 'semantics.html#the-hgroup-element', '&lt;hgroup&gt;')}}</td>
+    <td>{{Spec2('HTML WHATWG')}}</td>
+    <td></td>
   </tr>
- </tbody>
+  </tbody>
 </table>
 
+<h2 id="Accessibility concerns">Accessibility concerns</h2>
+
+<p>The presence of <code>hgroup</code> may remove information reported to assistive technology about the subheading portion of the heading group.
```

Comment:
```suggestion
<p>The presence of <code>hgroup</code> may remove information reported to assistive technology about the subheading portion of the heading group.</p>
```
(I’d personally be fine with not using explicit `</p>` end tags in MDN content, but I guess for consistency with existing content, this should have one.

Reply:
Oh shoot, my autocomplete didn't fire. Just pushed up a fix, thank you!